File: versions/java/Creatorinf.java
public class SimpleTool {
    // ANSI escape codes for color
    public static final String RESET = "\033[0m";  // Reset color
    public static final String BLUE_BACKGROUND = "\033[44m";
    public static final String GREEN_BACKGROUND = "\033[42m";
    public static final String WHITE_TEXT = "\033[37m";

    public static void main(String[] args) {
        // Change console background to blue and text to white
        System.out.print(BLUE_BACKGROUND + WHITE_TEXT);
        System.out.println("Thanks for downloading Simple Tool (on batch)");

        // Change console background to green and text to white
        System.out.print(GREEN_BACKGROUND + WHITE_TEXT);
        System.out.println("By Kaloyan-1206-Virt");

        // Reset to default console colors
        System.out.print(RESET);

        // Pause (wait for user to press Enter)
        System.out.println("Press Enter to continue...");
        try {
            System.in.read();
        } catch (Exception e) {
            e.printStackTrace();
        }

        // Exit the program
        System.exit(0);
    }
}

File: versions/C++/Creatorinf.cpp
#include <iostream>
#include <windows.h>

void setConsoleColor(int backgroundColor, int textColor) {
    HANDLE hConsole = GetStdHandle(STD_OUTPUT_HANDLE);
    CONSOLE_SCREEN_BUFFER_INFO consoleInfo;
    GetConsoleScreenBufferInfo(hConsole, &consoleInfo);

    // Set the background and text color
    SetConsoleTextAttribute(hConsole, (backgroundColor << 4) | textColor);
}

int main() {
    // Change console background to blue and text to white
    setConsoleColor(BLUE, FOREGROUND_WHITE);
    std::cout << "Thanks for downloading Simple Tool (on batch)" << std::endl;

    // Change console background to green and text to white
    setConsoleColor(GREEN, FOREGROUND_WHITE);
    std::cout << "By Kaloyan-1206-Virt" << std::endl;

    // Pause (wait for user to press Enter)
    std::cout << "Press Enter to continue..." << std::endl;
    std::cin.get();

    // Exit the program
    return 0;
}